osbuild/buildroot.py

```diff
@@ -80,7 +80,7 @@ class BuildRoot(contextlib.AbstractContextManager):
     only when exiting the context manager.
     """
 
-    def __init__(self, root, runner, libdir, var, *, rundir="/run/osbuild"):
+    def __init__(self, root, runner, libdir, var, *, rundir="/run/osbuild", caps = {}):
```

Comment:
The linter dose not like this default val for caps. Will think on alternatives

Reply:
Just drop this and initialise the field to the empty set.

Reply:
I went ahead an changed this to default as `None` for now but I can think of a number of options to solve this.
We can keep it as a defult of `None`, or, dont set `caps` at all and update the tests, or, move taking the capabilities to the `run` function.